homeassistant/components/media_player/plex.py

```diff
@@ -243,11 +242,30 @@ def update(self):
         self.update_devices(no_throttle=True)
         self.update_sessions(no_throttle=True)
 
+    # pylint: disable=no-self-use, singleton-comparison
+    def _convert_na_to_none(self, value):
+        """Convert PlexAPI _NA() instances to None."""
+        # PlexAPI will return a "__NA__" object which can be compared to
+        # None, but isn't actually None - this converts it to a real None
+        # type so that lower layers don't think it's a URL and choke on it
+        if value == None:
```

Comment:
[Docs for _NA](https://github.com/mjs7231/python-plexapi/blob/ba9eab0229e5e377c9cacc0eebc8dd05b5037216/plexapi/utils.py#L24-L33)

Seems like this is the "preferred" approach but I don't like it either. 

I would prefer this:

```python
from plexapi.utils import NA

if value is NA:
    …
```

You might want to cache the `NA` object on the instance in the constructor.